eventmesh-common/src/main/java/org/apache/eventmesh/common/config/ConfigMonitorService.java

```diff
@@ -52,20 +52,16 @@ public void load() {
                 if (configInfo.getObject().equals(object)) {
                     continue;
                 }
-
                 Field field = configInfo.getObjectField();
-                boolean isAccessible = field.isAccessible();
-                try {
-                    field.setAccessible(true);
-                    field.set(configInfo.getInstance(), object);
-                } finally {
-                    field.setAccessible(isAccessible);
-                }
+                field.setAccessible(true);
+                field.set(configInfo.getInstance(), object);
 
                 configInfo.setObject(object);
                 log.info("config reload success: {}", object);
             } catch (Exception e) {
                 log.error("config reload failed", e);
+            } finally {
+                configInfo.getObjectField().setAccessible(false);
```

Comment:
> The value of the property before the object was modified cannot be obtained here；Also, the value of the previous property is not necessarily false；What do you think?

Hi @eight-nines Here is the Java method description

![image](https://user-images.githubusercontent.com/15797831/218028626-250a9280-71ec-48bb-aed8-00b3a566cac1.png)
if there is a security manager, its checkPermission method is called with a ReflectPermission("suppressAccessChecks") permission. Instead of saying that the property becomes public when it's private.

Reply:
What I mean is here should save the last accessible status and recovery it after custom operation. But you uniformly set it to false. So I don't quite understand your intention of this modification, can you elaborate?

Reply:
Hi @eight-nines Can my example below help answer your question?

```java
public class Student {

    private int age;

    public String name;

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}
```
test class:
```java
public class Test {

    public static void main(String[] args) throws Exception{
        Student student = new Student();
        Field age = Student.class.getDeclaredField("age");
        try {
            age.set(student, 1);
        } catch (IllegalArgumentException e) {
            e.printStackTrace();
        } catch (IllegalAccessException e) {
            e.printStackTrace();
        }
        age.setAccessible(true);
        age.set(student, 1);
        Field name = Student.class.getDeclaredField("name");
        name.set(student, "mxsm");
        System.out.println(student.getName());
        name.setAccessible(false);
        name.set(student, "mxsm1");
        System.out.println(student.getName());
    }

}
```
run result:
![image](https://user-images.githubusercontent.com/15797831/218233791-9346ff96-0851-49f6-8bbd-26097c05c701.png)

explain:
Using the Student class as an example, the setAccessible method only sets a flag to tell the Java Virtual Machine whether to check if the property is private, public, or protected when executing. If set to true, it means no check is required, while false means a check is needed. Setting a non-public property to false using the setAccessible method does not change it to a private property. Therefore, it is okay to directly set it to false.

Reply:
Yes, after you change it, the original object will always become false, not the initial value; all, what if the initial value is true? And it will be used elsewhere, what should be done?

Reply:
This approach, destroying the original object, what do you think

Reply:
> This approach, destroying the original object, what do you think

@eight-nines  agree! since jdk9 this method is deprecated.  use canAccess is true. 
![image](https://user-images.githubusercontent.com/15797831/218241907-1ef00562-a460-4ea0-873c-a2c2d63798fc.png)